frontend/src/components/IstioWizards/K8sGatewaySelector.tsx

```diff
@@ -174,6 +186,20 @@ export class K8sGatewaySelector extends React.Component<Props, K8sGatewaySelecto
             )}
             {this.state.newGateway && (
               <>
+                {serverConfig.gatewayAPIClasses.length > 1 && (
```

Comment:
I guess another option would be to always display this so that the user can see the set value.  But this approach is fine as well, not a blocker.

Reply:
I made the Gateway Classes available in Debug info.